rotkehlchen/data_import/importers/binance.py

```diff
@@ -191,6 +191,7 @@ def are_entries(self, requested_operations: list) -> bool:
             (keys == {'Buy', 'Sell'} and counted['Buy'] % 2 == 0 and counted['Sell'] % 2 == 0) or
             (keys == {'Buy'} and counted['Buy'] % 2 == 0) or
             (keys == {'Sell'} and counted['Sell'] % 2 == 0) or
+            (keys == {'Buy', 'Sell'} and counted['Buy'] - counted['Sell'] == 0) or
```

Comment:
can you add a note in the ones that got deprecated that they aren't valid anymore? I believe we might want to keep them for compatibility but is good to note that is not the latest format